lua/lazy/help.lua

```diff
@@ -7,23 +7,30 @@ function M.index(plugin)
   if Config.options.readme.skip_if_doc_exists and vim.loop.fs_stat(plugin.dir .. "/doc") then
     return {}
   end
+  local files = vim.tbl_flatten(vim.tbl_map(function(file)
+    return vim.fn.expand(plugin.dir .. "/" .. file, false, true)
```

Comment:
You should also add `Util.norm(vim.fn.expan...)` to normalize the paths for windows

Reply:
`vim.fn.expand` will return a list. I'm not quite sure when to normalize, should I do it after expanding? Or before? 🤔

Reply:
after should be fine. Or just do it in the loop where you loop over the files, right before where you build the tag_filename

Reply:
Updated. I tested on linux/mac. But not very confident if filename generation would work correctly on Windows.